.gitignore

```diff
@@ -1,3 +1,4 @@
 .env
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+vendor
```

Comment:
Q: Should we add this?

Reply:
I think so, otherwise every time I commit Git, I need to pay attention to the changes in the vendor file. And similar to other mainstream warehouses, have been ignored.

This is the AWS example: https://github.com/aws/aws-sdk-go-v2/blob/main/.gitignore#L8